config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -12,9 +12,11 @@ dashboard_groups:
     - sig-node-node-feature-discovery
     - sig-node-node-problem-detector
     - sig-node-seccomp-operator
+    - sig-node-blocking
```

Comment:
Can we make it first?

Reply:
huh, I guess so. I didn't realize the order represented the actual order. lol